x/gov/migrations/v5/store.go

```diff
@@ -36,7 +36,7 @@ func MigrateStore(ctx sdk.Context, storeService corestoretypes.KVStoreService, c
 	}
```

Comment:
Note: this `MigrationStore` is registered [here](https://github.com/dydxprotocol/cosmos-sdk/blob/b48250d4e0efacbd40f3e0b980c1e1004d29e351/x/gov/module.go#L298), which is run in `RunMigrations` during the v_4_0_0_0 upgrade.